build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     gradlePluginPortal()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:3.6.1'
+    classpath 'com.android.tools.build:gradle:4.1.3'
```

Comment:
Gradle version should not be updated.

Reply:
ok sir I will make these changes

Reply:
PTAL while reverting the gradle the distribution url was changed is it fine or i need to revert that too.
![image](https://user-images.githubusercontent.com/64903405/119239376-3f6b7f80-bb66-11eb-85b2-86c464af1bb6.png)

@Victor-Titan

Reply:
@pravesh00 there should not be any changes in the gradle files.

Reply:
can I make a new PR ?